readthedocs/projects/tasks/search.py

```diff
@@ -95,7 +95,7 @@ def reindex_version(version_id, search_index_name=None):
 
     build_config = latest_successful_build.config or {}
     search_config = build_config.get("search", {})
-    search_ranking = search_config.get("ranking", [])
+    search_ranking = search_config.get("ranking", {})
```

Comment:
This is a bug, we expect this to be a dictionary. We aren't hitting this in production since all builds should have the defaults defined.